### PR TITLE
[releng] Update CHANGES with release v2022.07

### PR DESCRIPTION
**Pull request by PeterC-DLS**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -22,6 +22,13 @@ Highlights of each release are described below.  For more details, see our wiki
 which provides links to the Release Notes (itemized list of changes) for any release.
 
 
+v2022.07
+++++++++
+
+released *2022-08-02*
+
+This is a bugfix release. See notes on wiki: https://github.com/nexusformat/definitions/wiki/releasenotes_v2022.07
```
Comment (woutdenolf) [inline CHANGES.rst:30]: The link should have a double underscore:

https://github.com/nexusformat/definitions/wiki/releasenotes__v2022.07

**Review by woutdenolf — CHANGES_REQUESTED** [review]
Fix the link. Other than that all good.

File: CHANGES.rst (hunk 1 above)
Comment (PeterC-DLS) [reply CHANGES.rst]: Good spot and other URLs too

**Review by woutdenolf — APPROVED** [review]
lgtm

**Pull request merged**